### PR TITLE
Compute small probabilities in log domain.

### PR DESCRIPTION
**Pull request by krzysztofrusek**

### CODE REVIEW

File: mapc_sim/sim.py [hunk 1]
```diff
@@ -61,9 +61,11 @@ def network_data_rate(key: PRNGKey, tx: Array, pos: Array, mcs: Array, tx_power:
     sinr = sinr + tfd.Normal(loc=jnp.zeros_like(signal_power), scale=sigma).sample(seed=normal_key)
     sinr = (sinr * tx).sum(axis=0)
 
-    success_probability = tfd.Normal(loc=MEAN_SNRS[mcs], scale=2.).cdf(sinr) * (sinr > 0)
+    sdist = tfd.Normal(loc=MEAN_SNRS[mcs], scale=2.)
+    logit_success_prob = (sdist.log_cdf(sinr) - sdist.log_survival_function(sinr)) * (sinr > 0)
```
Comment (m-wojnar) [inline mapc_sim/sim.py]: Why do we need to subtract `log_survival_function`?

Comment (m-wojnar) [inline mapc_sim/sim.py]: I believe that the equivalent of multiptication by `sinr > 0` $\in \{0, 1\}$ in the probability domain should be multiplication by `-inf` if `sinr <= 0`, right?

Comment (krzysztofrusek) [reply mapc_sim/sim.py]: Becouse $logit(p)=log(p/1-p)=log(p)-log(1-p)$

Comment (krzysztofrusek) [reply mapc_sim/sim.py]: Yes

Comment (m-wojnar) [reply mapc_sim/sim.py]: True, you're right

**Review by m-wojnar — APPROVED** [review]

**Pull request merged**